tests/webapp/api/test_bugzilla.py

```diff
@@ -0,0 +1,49 @@
+import json
+import responses
+import re
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from rest_framework.test import APIClient
+
+def test_create_bug(webapp, eleven_jobs_stored):
+    """
+    test creating a bug in bugzilla
+    """
+
+    #def request_callback(request):
+    #    headers = {}
+    #    return(200, headers, {"success":"123456"})
+    #url_re = re.compile(r'https://bugzilla-dev.allizom.org/.*')
+    #responses.add_callback(
+    #    responses.POST, url_re,
+    #    callback=request_callback, content_type="application/json",
+    #)
+    #responses.add(
+    #    responses.POST, url_re,
+    #    body='{"success":"123456"}', status=200, content_type="application/json",
+    #)

```

Comment:
I'm having a really hard time figuring out how I'm supposed to be mocking out the requests to bugzilla. 

I've tried setting up a callback and using add_callback() to use it, but my test still seems to go out to the actual bugzilla-dev server, no matter what I put as the second parameter to add_callback().

Any pointers?